### PR TITLE
PageNotFound (404 Page) component and Storybook padding fixes

### PR DESCRIPTION
**Pull request by iandilling-adaptive**

### CODE REVIEW

File: packages/ui-kit-components/src/components/PageNotFound/PageNotFound.tsx [hunk 1]
```diff
@@ -0,0 +1,57 @@
+import { Box, Paper, Typography } from '@mui/material';
```
Comment (EdAdaptive) [inline packages/ui-kit-components/src/components/PageNotFound/PageNotFound.tsx:1]: Some of the elements on the page begin to overlap on smaller screen sizes.  
image

**Review by Honglin-Lu — APPROVED** [review]

File: packages/ui-kit-components/src/components/PageNotFound/PageNotFound.tsx (hunk 1 above)
Comment (iandilling-adaptive) [reply packages/ui-kit-components/src/components/PageNotFound/PageNotFound.tsx:1]: The Sadface icon might be too big for mobile screens, and the layout will need to change slightly. I'll check with Nate on what should be done here.

File: packages/ui-kit-components/src/components/PageNotFound/PageNotFound.tsx [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+    elevation={0}
+    square={true}
+  >
+    <Box sx={{ flex: '1 1 0', ...styles.container }}>
+      <Box>
+        <LogoLarge sx={styles.logo} />
+      </Box>
+      <SadFace sx={styles.sadFace} />
+      <Typography variant="h3" align="center" sx={styles.typography}>
+        Page not found!
```
Comment (EdAdaptive) [inline packages/ui-kit-components/src/components/PageNotFound/PageNotFound.tsx:68]: This text, and the paragraph beneath it, should not be capitalized according to the Figma design.  Might be able to add `text-transform: none` to the styles to fix this.

**Review by EdAdaptive — CHANGES_REQUESTED** [review]

File: packages/ui-kit-components/src/components/PageNotFound/PageNotFound.tsx (hunk 2 above)
Comment (iandilling-adaptive) [reply packages/ui-kit-components/src/components/PageNotFound/PageNotFound.tsx:69]: Good find. I set `text-transform: none` onto `styles.typography`

**Comment by iandilling-adaptive** [comment]
Nate came up with a stable responsive design for the 404 page this morning, and I have updated the PR with those changes. I included some Storybook padding fixes also, which is why the PR is so big.

**Review by EdAdaptive — APPROVED** [review]

**Pull request merged**